### PR TITLE
KOGITO-4540: Added extended use case to process-usertasks-quarkus example

### PR DESCRIPTION
**Pull request by nmirasch**
Added basic/extended versions to process-usertasks-quarkus example.
Moving current content we have at kogito-quickstarts/01-kogito-basic-example code and adding some screenshots.

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @pefernan Could you take a look here please, just to confirm this is what we need to get the getting starter as a extended versions. I have added some details and screen shots, just let me know what you think before follow with others.

File: process-usertasks-quarkus/basic/README.md [hunk 1]
```diff
@@ -64,14 +64,14 @@ NOTE: With dev mode of Quarkus you can take advantage of hot reload for business
 
 ```sh
 mvn clean package
-java -jar target/process-usertasks-quarkus-runner.jar
+java -jar ./target/process-usertasks-quarkus-basic-runner.jar
```
Comment (cristianonicolai) [inline process-usertasks-quarkus/basic/README.md]: I think the `./` is not really needed?

File: process-usertasks-quarkus/extended/docker-compose/persistence/hiring.proto [hunk 2]
```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
```
Comment (cristianonicolai) [inline process-usertasks-quarkus/extended/docker-compose/persistence/hiring.proto]: should this be generated instead?

**Review by cristianonicolai — COMMENTED** [review]
@nmirasch overall looks fine, just a few remarks, but is this a copy/move from the quickstart?

File: process-usertasks-quarkus/basic/README.md (hunk 1 above)
Comment (nmirasch) [reply process-usertasks-quarkus/basic/README.md]: agree!

File: process-usertasks-quarkus/extended/docker-compose/persistence/hiring.proto (hunk 2 above)
Comment (nmirasch) [reply process-usertasks-quarkus/extended/docker-compose/persistence/hiring.proto]: my fault, I added to repo when copy the directory!

**Comment by nmirasch** [comment]
@cristianonicolai This is try to bring the quickstart 01 basic example, adding some screenshots and testing..The point is to be sure this is what we want to see as a 'quick starter' included in a extended example to avoid that duplication.

**Comment by nmirasch** [comment]
Closing this PR regarding, I'm going to create another PR with the movement from kogito-quickstarts

**Pull request closed without merging**